pkg/jwt/decode_test.go

```diff
@@ -3,74 +3,111 @@ package jwt
 import (
 	"encoding/json"
 	"fmt"
+	"os"
 	"testing"
 )
 
-type DecodeTestInput struct {
+type DecodeTestCases struct {
```

Comment:
**suggestion:** This struct represents a singular test case. Rename to `DecodeTestCase`